app/components/Blockchain/Asset.jsx

```diff
@@ -1896,7 +1896,7 @@ class Asset extends React.Component {
                 render: item => {
                     return (
                         <FormattedAsset
-                            amount={item.amount}
+                            amount={item.amount * 10}
```

Comment:
?

Reply:
Collateral bid was being divided by 10 and not showing the correct bid amount and after changing this it started showing correct number.

Reply:
Perhaps better create another pull request for this.

Reply:
I think "being divided by 10" is somewhat arbitrary. The root cause is perhaps that BTS precision is 5 and USD precision is 4. So `* 10` here is likely wrong.

Reply:
> I think "being divided by 10" is somewhat arbitrary. The root cause is perhaps that BTS precision is 5 and USD precision is 4. So `* 10` here is likely wrong.

Makes sense, will do another PR for this and review it carefully.